src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -843,6 +843,27 @@ def playbook_artifact_replay(
                 return messages, exit_messages
         return messages, exit_messages
 
+    @staticmethod
+    @_post_processor
+    def pull_arguments(
+        entry: SettingsEntry,
+        config: ApplicationConfiguration,
+    ) -> PostProcessorReturn:
+        # pylint: disable=unused-argument
+        """Post process ```pull_arguments```
```

Comment:
Guessing this should be wrapped in 2 backticks (for ReST) instead of 3.

Reply:
thanks, got it.